core/contracts/financial_templates/FeePayer.sol

```diff
@@ -70,6 +70,22 @@ contract FeePayer is Testable {
         return regularFee.add(latePenalty);
     }
 
+    /**
+     * @notice Pays UMA DVM final fees to the Store contract. This is a flat fee charged for each price request.
+     * @return the amount of collateral that was paid to the Store.
+     */
+    function payFinalFees() internal returns (FixedPoint.Unsigned memory totalPaid) {
```

Comment:
nits: 
1. You may want to change the name of this function or the _payFinalFee() in the derived class so we don't get them confused.
2. Generally, we use a leading underscore for internal method names.

Reply:
should I `super` `_payFinalFees()`?

Reply:
I wouldn't. I think `super` should really only be used if you're trying to pass control up the inheritance tree iteratively. In this case, you just want to call the `_payFinalFees()` method on the FeePayer contract.

If you name them the same thing you'll be overriding that function, which would be confusing and not really necessary for this case. Note: it was useful for regular fees since the payFees() function can be called externally, and we wanted derived contracts to capture that call. Also, if you override the _payFinalFees() function, you'll also need to make sure you match the return types -- right now, I think they differ.

Reply:
FYI I did try `super` and all tests passed, but I do agree why `super` wouldn't make sense here. Changing the name now